qsirecon/interfaces/gradients.py

```diff
@@ -18,33 +18,201 @@
 LOGGER = logging.getLogger("nipype.interface")
 
 
-class RemoveDuplicatesInputSpec(BaseInterfaceInputSpec):
+class _GradientSelectInputSpec(BaseInterfaceInputSpec):
     dwi_file = File(exists=True, mandatory=True)
     bval_file = File(exists=True, mandatory=True)
     bvec_file = File(exists=True, mandatory=True)
+    b_file = File(exists=True, mandatory=True)
+    btable_file = File(exists=True, mandatory=True)
+    bval_distance_cutoff = traits.Float(5.0, usedefault=True)
```

Comment:
```suggestion
    bval_distance_cutoff = traits.Float(
        5.0,
        usedefault=True,
        desc='Distance from the center of each b-value cluster to consider within a given shell',
    )
```

Reply:
This is a different interface, which I don't believe is ever actually used in recon workflows. This particular distance cutoff is to determine whether two or more images are sampling approximately the same coordinate in q space.